### PR TITLE
Fix / householSize field everywhere

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

- Use good-looking householdSize field in home and student apartment insurance
- Ignoring accident for now, field is likely not needed there

## Checklist before requesting a review

- [x] I have performed a self-review of my code

### CODE REVIEW

File: apps/store/src/services/PriceForm/data/SE_HOUSE.ts [hunk 1]
```diff
@@ -129,13 +129,12 @@ export const SE_HOUSE: Template = {
...
           field: {
-            type: 'number',
-            min: 0,
-            max: 5,
+            type: 'householdSize',
             name: 'numberCoInsured',
-            label: { key: 'Number of co-insured' },
+            label: { key: 'Household size' },
             required: true,
             defaultValue: 0,
```
Comment (amandaflod) [inline apps/store/src/services/PriceForm/data/SE_HOUSE.ts:136]: Should we maybe change the default value to 1 since it is changed to household size?

**Review by amandaflod — APPROVED** [review]
Looks good! Just one little comment/question 🙂

File: apps/store/src/services/PriceForm/data/SE_HOUSE.ts (hunk 1 above)
Comment (alebedev) [reply apps/store/src/services/PriceForm/data/SE_HOUSE.ts:136]: It's still posted as numberCoInsured, I don't want to handle value transformation in this PR.  There's alternative approach to that in @robinandeer, let's discuss this with him first

Comment (amandaflod) [reply apps/store/src/services/PriceForm/data/SE_HOUSE.ts:136]: Ok, sounds good!

**Pull request merged**